extension/src/lambda.js

```diff
@@ -0,0 +1,27 @@
+const utils = require('./utils')
+const paymentHandler = require('../src/paymentHandler/payment-handler')
+const setup = require('./config/init/ensure-resources')
+
+const logger = utils.getLogger()
+let initialised = false
+
+exports.handler = async function(event) {
+  try {
+    if (!initialised) {
+      await setup.ensureCustomTypes()
```

Comment:
Why would you ensure only types here and not both types and API extension?

Reply:
In the case of the lambda, we don't want to ensure the API extension as that creates an HTTP API extension link in the commercetools project. For the lambda, it will never be invoked unless that link already exists, therefore making this ensureApiExtension part redundant.

Reply:
I see. So in case of Lambda usage it needs to be documented that Commercetoosl API extension needs to be created manually? Or you see any way to automate that?

Reply:
So we just had another discussion about this, and we realised that it probably doesn't make sense to create these resources within the lambda init at all. The issue is that these resources all need to exist before the lambda is invoked, so it probably makes sense to not have this ensureCustomTypes step here at all. We weren't actually planning on using this anyway as we will be managing all of our custom objects using terraform with this provider from [here](https://github.com/labd/terraform-provider-commercetools). If we were to remove this step from here, we could just document that if using the lambda, you have to create the resources yourself or using some other kind of automation (such as the terraform provider above). What are your thoughts on this?

Reply:
@butenkor Do you have any thoughts on the above?

Reply:
> So we just had another discussion about this, and we realised that it probably doesn't make sense to create these resources within the lambda init at all. The issue is that these resources all need to exist before the lambda is invoked, so it probably makes sense to not have this ensureCustomTypes step here at all.

Agree.

> If we were to remove this step from here, we could just document that if using the lambda, you have to create the resources yourself or using some other kind of automation (such as the terraform provider above). What are your thoughts on this?

Maybe for convenience we could add terraform template (for custom types and API extension) here and a short guide how to run it?

Alternatively it could be just a simple function which can be run from command line and create required resources (due to complexity updates on the type definition would not be supported for now).

Reply:
I have now removed the ensureCustomTypes from the lambda.

I have create a seperate npm command that can be used to invoke the ensureCustomTypes method from ensure-resources.js. If provided with the CTP project key, client id and secret, it will create the custom types in the project. I have also added this detail to the documentation under the lambda section.

Reply:
Besides the 1 comment i left in one of the latest your commits, looks good. Thanks!